community/gardening/dev-environment.md

```diff
@@ -0,0 +1,222 @@
+---
+layout: single
+title:  "Development Environments for Spinnaker Gardening Days"
+sidebar:
+  nav: community
+---
+What's the best environment Spinnaker development? How can you set up your workstation to debug a Spinnaker service?  It depends! Consider these constraints when choosing your strategy:
+
+* __Locally available computing resources__: hosting Spinnaker services is memory-intensive.
+* __Access and cost management for public clouds__: Spinnaker can be hosted in the cloud, where you'll pay for compute.
+* __Familiarity with Kubernetes__: you may use Kubernetes tools to manage your Spinnaker environment if you prefer.
+
+# Install Spinnaker
+First things first, to develop Spinnaker, you'll need a Spinnaker instance. To get that, you have options:
+* Install [Minnaker](https://github.com/armory/minnaker), a POC Spinnaker instance that runs in a Linux VM on your local machine, or in the cloud.
+* [Clone and install each Spinnaker service locally.](#classic-local-installation-method)
+* [Install Spinnaker to your Kubernetes cluster](#kubernetes-installation-methods), running in your cloud provider or private cloud of choice. Read more about the [Kubernetes & Docker method](#kubernetes-and-docker-method) below.
+
+## Classic local installation method
+Follow the [Getting Set Up](https://www.spinnaker.io/guides/developer/getting-set-up/) guide to install Spinnaker locally.
+
+## Minnaker method
+If you're not sure which method to choose, we suggest following [these instructions to install Minnaker](https://github.com/armory/minnaker) in your chosen environment. This simplifies installation steps, and uses Kubernetes (K3S) under the hood. Install in a cloud VM or a local Ubuntu 18.04 VM.
+
+### Set up local development environment
```

Comment:
Recommend numbers instead of bullets here.

Reply:
Tried this but it is hard with the nested lists. Can we figure out a better number strategy in a later PR?